src/platform/Zephyr/BLEManagerImpl.h

```diff
@@ -105,6 +105,8 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
 #if CHIP_ENABLE_ADDITIONAL_DATA_ADVERTISING
     PacketBufferHandle c3CharDataBufferHandle;
 #endif
+    // The summarized number of Bluetooth LE connections related to the device (including these not related to Matter service).
+    uint16_t mBleConnNum;
```

Comment:
nit: could you better name `mGAPConns` to distinguish it from `mBleConnNum`. Maybe `mMatterConnNum` vs `mTotalConnNum`?